packages/postgres-database/src/simcore_postgres_database/models/products_prices.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sa
+
+from ._common import NUMERIC_KWARGS, column_created_datetime
+from .base import metadata
+from .products import products
+
+#
+# - Every product has an authorized price
+# - The price is valid from the creation date until a new price is created
+# - No rows are deleted!
+# - If a product has no price, it is assumed zero
+#
+
+products_prices = sa.Table(
```

Comment:
Didn't you want to go in a similar fashion as we have it in the for example pricing plans: meaning we have two columns valid_from and valid_to? so we can nicely have all the history?

Reply:
Also that way it would be easier/more effective to query the current active price for the product

Reply:
I did the renaming. We discussed pros/cons about this offline.